modules/creating-serverless-apps-yaml.adoc

```diff
@@ -16,13 +16,13 @@ To create a serverless application by using YAML, you must create a YAML file th
 apiVersion: serving.knative.dev/v1
 kind: Service
 metadata:
-  name: event-delivery
+  name: hello
   namespace: default
 spec:
   template:
     spec:
       containers:
-        - image: quay.io/openshift-knative/knative-eventing-sources-event-display:latest
+        - image: docker.io/openshift/hello-openshift
```

Comment:
again, this is now inconsistent with the rest of docs, it would be better to update the quay.io image if that's possible https://docs.openshift.com/container-platform/4.7/serverless/event_sources/serverless-pingsource.html

upstream also uses `event-delivery` as the generic service for examples, so maybe we should start with standardizing it there first?